nimbus.fml.yaml

```diff
@@ -299,34 +299,50 @@ features:
             "inactive-tabs": true,
           }
   contextual-hint-feature:
-    description: This set holds all features pertaining to the usage of contextual hints, whether it's to introduce new features or remind users of existing ones.
+    description: This set holds contextual hints that can be toggled to hide or show based on these settings.
     variables:
       features-enabled:
         description: This property provides a lookup table of whether specific contextual hints are enabled.
         type: Map<ContextualHint, Boolean>
         default:
         {
-          "toolbar-contextual-hint": true,
-          "jump-back-in-contextual-hint": true,
           "jump-back-in-synced-tab-contextual-hint": false
         }
     defaults:
       - channel: beta
         value: {
           "features-enabled": {
-            "toolbar-contextual-hint": true,
-            "jump-back-in-contextual-hint": true,
```

Comment:
Question: I don't remember exactly, but is the intent that we can toggle CFRs on or off remotely with an experiment? (Double check with @dnarcese, perhaps to make sure). If so, these should remain in here, and simply be used as a check for whether or not a specific CFR can be shown. Then, the copy variable (see my other comment) will be the one that controls whether or not we show new copy.

Reply:
Only the Synced tab CFR needs to be hidden behind a feature flag now. And there's no plan (that i've heard of) to have the others toggleable.

Reply:
Correct. Only synced tab CFR needs to be on/off. The others only need old/new copy.